server/api/sessions.js

```diff
@@ -84,6 +84,35 @@ internals.applyRoutes = function (server, next) {
 
     server.route({
         method: 'DELETE',
+        path: '/sessions/my/{id}',
+        config: {
+            auth: {
+                strategy: 'simple',
+                scope: 'account'
+            }
+        },
+        handler: function (request, reply) {
+
+            const id = request.auth.credentials.roles.account._id.toString();
```

Comment:
I'm not sure how we're finding a session by the `account` id of the user. What if they're an `admin`?

I think you meant to use the `id` url param from above?

Reply:
Yes